parse-owasp-zap-xml.py

```diff
@@ -17,12 +17,14 @@
         for alert in site.findall('.//alertitem'):
             id = '{}, CWE id {}, WASC id {}, Risk {}, Plugin ID {}'.format(alert.find('name').text,
                                                     alert.find('cweid').text,
-                                                    alert.find('wascid').text,
+                                                    alert.findtext('wascid',"None"),
```

Comment:
Nice catch - should the rest of these also be switched or are you confident that they'll be there, or this should raise an error if they're not found?

Reply:
Next time it happens I'll change all of them. I hope to actually retire this with a real tool like DefectDojo, so not wanting to over-invest there.